Algorithms/Easy/9_PalindromeNumber/Solution.py

```diff
@@ -0,0 +1,8 @@
+class Solution(object):
+    def isPalindrome(self, x): # type x : int & return type: bool
+        if x < 0:
+            return False
+        b =int(str(x) [::-1])
```

Comment:
Change this code to 
```py
b = int(str(x) [::-1])
```

Reply:
> Hi @FahimFBA
> Please resolve the requested changes, then your pr will be ready for merge.
> 
> Cheers 🎉

Hi @Tahanima , I have resolved the requested changes. 😀